src/app/listingpage/productcard/productcard.component.css

```diff
@@ -0,0 +1,17 @@
+i{
```

Comment:
* do not use i tag
* use classes to style - not element selectors

Reply:
okay